src/main/cpp/util/BUILD

```diff
@@ -196,23 +196,24 @@ filegroup(
     visibility = ["//visibility:public"],
 )
 
-pkg_tar(
-    name = "cpp_util_with_deps_archive",
-    extension = "tar.gz",
-    mode = "755",
-    visibility = ["//third_party/ijar:__pkg__"],
-    deps = [
-        ":cpp_util_archive",
-        "//src/main/native/windows:src_main_native_windows",
+# A zip containing source files used by ijar.
+genrule(
+    name = "cpp_util_with_deps_zip",
+    srcs = [
+        ":cpp_util_zip",
+        "//src/main/native/windows:native_windows_zip",
     ],
+    outs = ["cpp_util_with_deps.zip"],
+    cmd = "$(location //third_party:merge_zip_files.sh) nodir $(OUTS) $(SRCS)",
```

Comment:
nit: `$(OUTS)` allows multiple outputs, but the script expects just one. Use `$<`, so if another `outs` entry is added, the rule breaks.

Also in other genrules.

Reply:
TIL about `$<` and `$@`, thanks! In this case `$@` is needed (`$<` checks the inputs).